Jenkinsfile

```diff
@@ -305,12 +305,12 @@ def buildPerformance() {
           sh 'unzip core-benchmarks.zip -d test/bench/'
           sh 'rm core-benchmarks.zip'
 
-          sh '''
+          sh """
             cd test/bench
             mkdir -p core-benchmarks results
             ./gen_bench_hist.sh
             ./parse_bench_hist.py --local-html results/ core-benchmarks/
-          '''
+          """
```

Comment:
If I understand this correctly you are changing the quoting from strong to weak, but there is seemingly nothing to be interpreted inside, so wouldn't it be better to keep it strong? Maybe a n00b question. 😄